tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationAttachmentPointTest.java

```diff
@@ -82,59 +82,59 @@ public void testInvalidAttachmentOnObjectMethodDefinition() {
         int index = 25;
         int line = 117;
         validateError(compileResult, index++, "annotation 'v1' is not allowed on object_method, function",
-                      line, 5);
+                line, 5);
```

Comment:
Shall we revert these changes if they are not needed? Should wee add that missing one too?